py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -0,0 +1,74 @@
+from .config_base import ConfigType
+from collections.abc import Iterable
+from pathlib import Path
+
+
+class Batch(object):
+    """
+    will store a collection of filenames that should be downloaded and converted
+    from json into parquet.
+    """
+    def __init__(self, config_type: ConfigType) -> None:
+        self.config_type = config_type
+        self.filenames = []
+        self.total_size = 0
+
+    def __str__(self) -> None:
+        return "%d bytes across %d files of type %s" % (self.total_size,
+                                                        len(self.filenames),
+                                                        self.config_type)
+
+    def add_file(self, filename: str, filesize: int) -> None:
+        self.filenames.append(filename)
+        self.total_size += filesize
+
+    def trigger_lambda(self) -> None:
+        raise Exception("not impled yet")
+
+    def create_event(self) -> dict:
+        return {
+            'bucket':'mbta-gtfs-s3',
+            's3_files': self.filenames,
+        }
```

Comment:
small helper class that will collect files to ingest together, keep track of total filesize, and create event json.